packages/backend/src/graphql/resolvers/liquidity.ts

```diff
@@ -440,13 +440,25 @@ export const depositOutgoingPaymentLiquidity: MutationResolvers<ApolloContext>['
     args,
     ctx
   ): Promise<ResolversTypes['LiquidityMutationResponse']> => {
+    const telemetry = await ctx.container.use('telemetry')
+    const stopTimer = telemetry.startTimer(
+      'deposit_outgoing_payment_liquidity_ms',
+      {
+        callName: 'depositOutgoingPaymentLiquidity'
+      }
+    )
     const { outgoingPaymentId } = args.input
     const webhookService = await ctx.container.use('webhookService')
+    const stopTimerWh = telemetry.startTimer('wh_get_latest_ms', {
+      callName: 'webhookService.getLatestByResourceId'
+    })
     const event = await webhookService.getLatestByResourceId({
       outgoingPaymentId,
       types: [OutgoingPaymentDepositType.PaymentCreated]
     })
+    stopTimerWh()
     if (!event || !isOutgoingPaymentEvent(event)) {
+      stopTimer()
```

Comment:
Something we don't necessary have to tackle in this PR, but what can we do when it comes to stopping timers during a happy path vs an error? is there something we can add to make that distinction in the metrics?

Reply:
Yeah we could have a `success: boolean` attribute on the underlying histogram. I was inspired by a pattern that was shown to me, and it had that.

Reply:
Not used currently, but it should be possible now. I updated the closure returned from `startTimer` to accept `additionalAttributes` which will be merged on the ones initially provided to `startTimer`. Usage example added to tests (and tests added for `startTimer` in general - noticed we didnt have any).